lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -56,17 +53,27 @@ def self.localize_project
         #       in the release-toolkit instead, and move this code away from `ios_git_helper`.
         #
         def self.update_metadata
-          Action.sh("cd #{ENV['PROJECT_ROOT_FOLDER']} && ./Scripts/update-translations.rb")
+          Action.sh("cd #{get_from_env!(key: 'PROJECT_ROOT_FOLDER')} && ./Scripts/update-translations.rb")
 
-          strings_files = Dir.chdir(File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'])) do
-            Dir.glob('*.lproj/*.strings')
-          end
           Fastlane::Helper::GitHelper.commit(message: 'Update translations', files: strings_files, push: false)
 
           Action.sh('cd fastlane && ./download_metadata.swift')
 
           Fastlane::Helper::GitHelper.commit(message: 'Update metadata translations', files: './fastlane/metadata/', push: true)
         end
+
+        def self.strings_files
+          project_root = get_from_env!(key: 'PROJECT_ROOT_FOLDER')
+          project_name = get_from_env!(key: 'PROJECT_NAME')
+
+          Dir.glob(File.join(project_root, project_name, '**', '*.strings'))
+        end
+
+        def self.get_from_env!(key:)
+          ENV.fetch(key) do
+            UI.user_error! "Could not find value for \"#{key}\" in environment."
+          end
+        end
```

Comment:
This is not the nicest method, and it's probably duplicated somewhere in the library.

As a followup, I think we may want to add this to `ENV` directly and find a nice name (`fetch!`?) for it and make it flexible.

Reply:
How does the output look if you just use `ENV.fetch` without a block? According to the ruby documentation, `fetch` will raise a `KeyError` if you don't provide a block nor default value; wondering if Fastlane doesn't already catch that raised `KeyError` and prints a nice message for it already instead of you having to handle the `UI.user_error!` explicitly here?

Reply:
(That being said I agree to ship this as is and circle back on this detail later if needs be)